django/core/management/commands/shell.py

```diff
@@ -111,10 +115,52 @@ def python(self, options):
         # Start the interactive interpreter.
         code.interact(local=imported_objects)
 
+    def get_and_report_namespace(self, verbosity):
+        namespace = self.get_namespace()
+
+        if verbosity == 1:
+            self.stdout.write(
+                f"{len(namespace)} objects imported automatically",
```

Comment:
I would find this more clear (and also more reassuring if I hadn't heard about this feature and was a bit wary):
```suggestion
                f"{len(namespace)} models imported automatically",
```

Reply:
I guess this is here to anticipate customizations that import other things, but if that's so, then perhaps we should consider exposing this in a class attribute or something easily overridden to support customization without too much trouble, but without muddying the main use case.

Reply:
Oh, I wasn't up to speed on ticket-35680. So we should leave this as is.